src/test/java/com/vi/counselingtoolsservice/api/facade/ToolsFacadeTest.java

```diff
@@ -72,9 +70,8 @@ public void getAssignedTools() {
     App app3 = new App();
     app3.setType("CONSULTANT_APP");
     apps.add(app3);
-    budibaseApps.setData(apps);
     Mockito.when(budibaseApiService.getBudibaseUser(adviceSeekerId)).thenReturn(budibaseUser);
-    Mockito.when(budibaseApiService.getApps()).thenReturn(budibaseApps);
+    Mockito.when(budibaseApiService.getApps()).thenReturn(apps);
```

Comment:
not enough coverage for the provided code changes

Reply:
add few tests and make sense to have them